life/modernized/19-chapter-19.adoc

```diff
@@ -310,13 +315,21 @@ Farewell, my dear friend, and the Lord be with you!
 
 I am, more than I can say, your great lover and respectful friend,
 
+[.signed-section-signature]
 William Penn.
 
+[.postscript]
 P+++.+++ S. I refer you to the enclosed for passages.
 We visited Giftall and Hooftman, and they us.
 They were at one or two of the meetings at Amsterdam.
 Vale in aternum!
 
+--
+
+[.embedded-content-document.letter]
+--
+
+[.letter-heading]
 For Anna Maria de Homes, styled Countess of Homes.
 
 Jesus be with your spirit.
```

Comment:
This is really weird, but I guess this phrase "Jesus be with your spirit" should maybe be part of the letter heading. I noticed that the letter above this had something funny like this in the letter heading ("Salvation in the cross. Amen"), and I don't know where else to put it. 

Also... Github won't let me leave a comment on line 97, but there is a bracketed comment +++[+++says W. P] WITHIN a footnote (which is also put within brackets). Whenever this happens it causes the footnote to mess up, because the first close-bracket ends the footnote before it is really done. So, here we just need to change the brackets to parenthesis... I mean, where it says **+++[+++says W. P]** change it to just **(says W. P.)**. This will need to be done in the modernized and the original. Hope that makes sense.